### PR TITLE
docs(aio): document new Router events and fix sequence of route events

### PR DESCRIPTION
**Pull request by jasonaden**
Add doc for new Route level events

### CODE REVIEW

File: aio/content/guide/router.md [hunk 1]
```diff
@@ -243,23 +266,68 @@ During each navigation, the `Router` emits navigation events through the `Router
...
+    </td>
+  </tr>
+
+  <tr>
+    <td>
+      <code>ChildActivationEnd</code>
+    </td>
+    <td>
+
+      An [event](api/router/ChildActivationEnd) triggered when the Router finishes activaing a child route. This event
```
Comment (gkalpak) [inline aio/content/guide/router.md]: _activaing_ --> _activating_

File: aio/content/guide/router.md [hunk 2]
```diff
@@ -243,23 +266,68 @@ During each navigation, the `Router` emits navigation events through the `Router
...
 
   <tr>
     <td>
-      <code>RouteConfigLoadEnd</code>
+      <code>ChildActivationStart</code>
     </td>
     <td>
 
-      An [event](api/router/RouteConfigLoadEnd) triggered after a route has been lazy loaded.
+      An [event](api/router/ChildActivationStart) triggered when the Router begins activaing a child route. This event
```
Comment (brandonroberts) [inline aio/content/guide/router.md]: I think it should say `activating each child route`

**Review by brandonroberts — COMMENTED** [review]
One minor nit, but everything else LGTM

File: aio/content/guide/router.md (hunk 1 above)
Comment (jasonaden) [inline aio/content/guide/router.md]: fixed

File: aio/content/guide/router.md (hunk 2 above)
Comment (jasonaden) [inline aio/content/guide/router.md]: So I changed the wording to `begins activating a route's children.` This is actually most accurate as the event will fire once no matter how many children are being activated.

**Review by brandonroberts — APPROVED** [review]

**Comment by jasonaden** [comment]
Closing as this has been superseded by another commit

**Pull request closed without merging**